Makefile

```diff
@@ -38,6 +40,11 @@ update-deps:
 	# see http://sed.sourceforge.net/sed1line.txt
 	find vendor -type f -exec sed -i -e :a -e '/^\n*$$/{$$d;N;ba' -e '}' "{}" \;
 
+install:
+	install -D -m 755 oci-create-runtime-bundle ${INSTALL_DIR}
+	install -D -m 755 oci-unpack ${INSTALL_DIR}
+	install -D -m 755 oci-image-validate ${INSTALL_DIR}
```

Comment:
I'm not excited about repeatedly listing all of our compiled tools.  I don't see a way around that here, but with #28 we could do:

    install: $(TOOLS)
        install -D -t $(INSTALL_DIR) $^